pyproject.toml

```diff
@@ -120,10 +119,11 @@ force-exclude = """
 )
 """
```

Comment:
I think this `force-exclude` should now be `extend-exclude = ["nibabel/externals"]`